packages/autoloader/src/AutoloadGenerator.php

```diff
@@ -244,6 +260,56 @@ private function getAutoloadClassmapPackagesFile( $classMap ) {
 INCLUDE_CLASSMAP;
 	}
 
+	/**
+	 * Take the autoloads array and return the fileMap that contains the path and the version for each namespace.
+	 *
+	 * @param array      $autoloads Array of autoload settings defined defined by the packages.
+	 * @param Filesystem $filesystem Filesystem class instance.
+	 * @param string     $vendorPath Path to the vendor directory.
+	 * @param string     $basePath Base Path.
+	 *
+	 * @return string $fileMap
+	 */
+	private function getFileMap( array $autoloads, Filesystem $filesystem, $vendorPath, $basePath ) {
+		$fileMapString = '';
+		foreach ( $autoloads['files'] as $file_id => $package ) {
+			$key       = var_export( $file_id, true );
+			$pathCode        = $this->getPathCode( $filesystem, $basePath, $vendorPath, $package['path'] );
+			$versionCode     = var_export( $package['version'], true );
```

Comment:
Can we please have all equals signs align here?

Reply:
Done in 4bf8ca0